canonical/ubuntu.com
# fix for peer score coming as string

## Done

- Fixes NPS scores coming in as string on sheets

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- fill the form in /credentials/exit-survey
- Check the sheet to check if the NPS scores land as integers instead of string

## Issue / Card

Fixes #[CRED-333](https://warthogs.atlassian.net/browse/CRED-333?atlOrigin=eyJpIjoiZWQ4OTA4Yjk2OThlNDg3MmEzY2E3ZDEwYjBiYmNkYzciLCJwIjoiaiJ9)

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)


[CRED-333]: https://warthogs.atlassian.net/browse/CRED-333?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
